tests/common/utilities.py

```diff
@@ -1151,20 +1153,24 @@ def _paramiko_ssh(ip_address, username, passwords):
             ssh.connect(ip_address, username=username, password=password,
                         allow_agent=False, look_for_keys=False, timeout=10)
             return ssh, password
-        except paramiko.AuthenticationException:
+        except AuthenticationException:
             continue
         except Exception as e:
             logging.info("Cannot access device {} via ssh, error: {}".format(ip_address, e))
             raise e
     logging.info("Cannot access device {} via ssh, error: Password incorrect".format(ip_address))
-    raise paramiko.AuthenticationException
+    raise AuthenticationException
 
 
 def paramiko_ssh(ip_address, username, passwords):
     ssh, pwd = _paramiko_ssh(ip_address, username, passwords)
     return ssh
 
 
-def get_dut_current_passwd(ip_address, username, passwords):
-    _, pwd = _paramiko_ssh(ip_address, username, passwords)
-    return pwd
+def get_dut_current_passwd(ipv4_address, ipv6_address, username, passwords):
+    try:
+        return _paramiko_ssh(ipv4_address, username, passwords)
+    except (AuthenticationException, NoValidConnectionsError, socket.timeout) as e:
```

Comment:
This block catches too many exceptions, which definitely works, but not that clear and efficient enough, suggest(please verify the syntax, I didn't test it)：

```
    except AuthenticationException:
        raise
    else: 
        return _paramiko_ssh(ipv6_address, username, passwords)
```
If we catch Authentication error, it means the connection by IPv6 is good, no need to retry with IPv6

Reply:
Updated, thanks for your suggestion

Reply:
//comment fixed